index.md

```diff
@@ -16,98 +16,91 @@ order: 1
 ## Introduction
 {:#introduction}
 
-Web accessibility evaluation tools are software programs or online services that help determine if a Web site is accessible. This document highlights features of evaluation tools, and discusses different aspects to consider when selecting these tools. Links to different tools are provided in the [Web Accessibility Tool List](https://www.w3.org/WAI/ER/tools/).
+Web accessibility evaluation tools are software programs or online services that help determine if a website is accessible. This document highlights features of evaluation tools, and discusses different aspects to consider when selecting these tools. Links to different tools are provided in the [Web Accessibility Evaluation Tools List](https://www.w3.org/WAI/ER/tools/).
 
 ## What to Expect from Evaluations Tools
 {:#Expect}
 
-Web accessibility evaluation tools can help you quickly identify potential accessibility issues. You can use them through all phases of the web design and development process. Tools can provide fully-automated checks, as well as help you with manual review.
+Web accessibility evaluation tools can help you quickly identify potential accessibility issues. You can use them through all phases of the web design and development process. Tools provide fully-automated checks, as well as help you with manual review.
 
 But we cannot check all accessibility aspects automatically. Human judgement is often required. Sometimes evaluation tools can produce false or misleading results. Web accessibility evaluation tools can not *determine* accessibility, they can only *assist* in doing so.
 
 ## Features of Evaluation Tools
 {:#Features}
 
-Web accessibility evaluation tools target different audiences. This includes designers, developers, non-technical content authors, quality assurance testers, and sometimes also end-users. Tools tend to offer different features and functionality which may help users to compare and assess Web accessibility evaluation tools for their specific needs.
```

Comment:
Web was capitalized, as per the style guide, though I have no strong objection to lowercase - which is provided for in the style guide.

Reply:
I thought it is capitalized when we talk about "the Web" but lower case otherwise, like "web accessibility" and "web pages". But I don't feel strongly either.

Reply:
+1 style guide has it lowercase as adjective https://www.w3.org/WAI/EO/wiki/Style#Capitalization